### ISSUE TITLE
Two Games running at once after update pushed

### ISSUE THREAD

**Issue by THCue** [prom3theu5/ConanExilesServerUpdater#24]
After an update not sure if server is shutting just but it shows starting a new task of the game ie two games running at the same time. Have to kill off them both and let it restart.

some logs and errors I seem to get.

OS is Windows 2012R2

The description for Event ID 1 from source ConanSandbox-PID3808 cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

**Comment by THCue** [comment]
Also noticed the game not doing a full patch not sure what happen. Had to manually patch and rerun everything.

https://gyazo.com/50368da7d4b1466e53b82c00a8da67c9

Just not shutting down it seems.

{
  "Twitch": {
    "OAuth": "",
    "Username": "",
    "Channel": "0"
  },
  "Discord": {
    "DiscordToken": "Mjg1MjA3NTIzNTc1OTg4MjI1.C5OzuQ.ERr84shgF-nTDKthYpNxxxSq750",
    "ChannelId": 207642415119xxxx69
  },
  "Conan": {
    "FolderPath": "c:\\Exiles\\",
    "Executable": "ConanSandboxServer.exe",
    "StartupParameters": "",
    "RaidingProtectionHoursEnabled": false,
    "RaidingStartHour": 0,
    "RaidingLengthInHours": 0,
    "AvatarsEnabledCertainHours": false,
    "AvatarActivationHour": 0,
    "AvatarsActiveLengthInHours": 0
  },
  "Update": {
    "ShouldInstallSteamCmdIfMissing": false,
    "ShouldInstallConanServerIfMissing": false,
    "SteamCmdPath": "c:\\SteamCMD\\",
    "AnnounceDiscord": true,
    "AnnounceTwitch": false,
    "AnnounceMinutesBefore": 12,
    "UpdateCheckInterval": 5,
    "InstalledBuild": 1668495,
    "UpdateOnLaunch": true
  },
  "General": {
    "ShouldRestartConanOnNotRunning": true,
    "RestartServerAfterHours": 0
  }
}

**Comment by moveit124** [comment]
I am also having this issue, with the patch before today's I had to manually update.  I also heard that because the update failed some servers had to completely wipe due to corruption.  Today's patch also did not restart the client properly, but I did not need to manually update.

**Comment by THCue** [comment]
Yep seems it does not want to shutdown the old instance even after the update and runs two at the same time forcing it to read invalid within the game.

**Comment by DJThorTX** [comment]
I am having the exact same problem over here, I hope a solution is found.

**Comment by arimaze** [comment]
Hi, 

same issue, 2 instances. I think, it should be good to check number of instance running.

**Comment by DJThorTX** [comment]
same problem again. 2 instances

**Comment by THCue** [comment]
Not sure if Prom is working on this project anymore. I use CAT and Mod it you can check it out at http://www.jffgaming.com/ just a heads up I know how you guys feel.

**Comment by arimaze** [comment]
Hi,

Thank you for sending CAT link . I installed it it works well.

DO you have some idea how to automatically update conan exiles mod  ?

**Comment by THCue** [comment]
You can Message me at http://www.jffgaming.com/ as THCue am the CAT MOD
